pkgs/top-level/aliases.nix

```diff
@@ -1182,7 +1182,7 @@ mapAliases ({
     ocamlPackages_latest;
 
   mumble_git = pkgs.mumble; # added 2019-08-01
-  murmur_git = pkgs.murmur; # added 2019-08-01
+  mumble-server_git = pkgs.mumble-server; # added 2019-08-01
```

Comment:
```suggestion
  murmur_git = pkgs.mumble-server; # added 2019-08-01
  murmur = pkgs.mumble-server; # added 2021-02-09
```

Reply:
Shouldn't we instead deprecrate the alias with a throw?

Reply:
also fine by me